pkg/pillar/kubeapi/nokube.go

```diff
@@ -6,15 +6,16 @@
 package kubeapi
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/lf-edge/eve/pkg/pillar/base"
 	"github.com/lf-edge/eve/pkg/pillar/pubsub"
 )
 
 // WaitForKubernetes in this file is just stub for non-kubevirt hypervisors.
-func WaitForKubernetes(
-	string, *pubsub.PubSub, *time.Ticker) error {
+func WaitForKubernetes(string, *pubsub.PubSub, *time.Ticker,
```

Comment:
I don't know the context of this code, but its name reflects my entire feeling regarding the work on EVE now: "Wait For Kubernetes"... =D

Sorry for the off-topic, do not consider it as a review comment =)